From db6d15bf12a0123e4320e5fd7cb688331dea1bdc Mon Sep 17 00:00:00 2001
From: Marcelo Ricardo Leitner <marcelo.leitner@gmail.com>
Date: Fri, 24 Aug 2018 09:52:58 -0300
Subject: [PATCH 1/3] build: remove v4.12 secondary defines in favor of
 HAVE_SCTP_STREAM_RECONFIG

These were backups, commented out since beginning.
HAVE_SCTP_STREAM_RECONFIG is enough to identify that these are there, so
lets use only one.

Signed-off-by: Marcelo Ricardo Leitner <marcelo.leitner@gmail.com>
Reviewed-by: Xin Long <lucien.xin@gmail.com>
---
 configure.ac                  | 8 --------
 src/include/netinet/sctp.h.in | 2 --
 2 files changed, 10 deletions(-)

diff --git a/configure.ac b/configure.ac
index 2ae36ec..dad658c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -82,14 +82,6 @@ AC_CHECK_FUNCS([bzero gethostbyname gettimeofday memmove memset select socket \
 LKSCTP_CHECK_MEMBER([struct sctp_event_subscribe.sctp_stream_reset_event],
 		    [HAVE_SCTP_STREAM_RESET_EVENT])
 
-# Support for assoc reset event, added on v4.12, c95129d127c6
-#LKSCTP_CHECK_MEMBER([struct sctp_event_subscribe.sctp_assoc_reset_event], \
-#		    [HAVE_SCTP_ASSOC_RESET_EVENT])
-
-# Support for stream change event, added on v4.12, b444153fb5a6
-#LKSCTP_CHECK_MEMBER([struct sctp_event_subscribe.sctp_stream_change_event], \
-#		    [HAVE_SCTP_STREAM_CHANGE_EVENT])
-
 # RFC 6525 (Stream Reconf), finished on v4.12, c0d8bab6ae51
 LKSCTP_CHECK_DECL([SCTP_RECONFIG_SUPPORTED], [HAVE_SCTP_STREAM_RECONFIG])
 
diff --git a/src/include/netinet/sctp.h.in b/src/include/netinet/sctp.h.in
index c049077..2009f1c 100644
--- a/src/include/netinet/sctp.h.in
+++ b/src/include/netinet/sctp.h.in
@@ -61,8 +61,6 @@ extern "C" {
 #define HAVE_SCTP_CANSET_PRIMARY
 
 #undef HAVE_SCTP_STREAM_RESET_EVENT
-#undef HAVE_SCTP_ASSOC_RESET_EVENT
-#undef HAVE_SCTP_STREAM_CHANGE_EVENT
 #undef HAVE_SCTP_STREAM_RECONFIG
 #undef HAVE_SCTP_PEELOFF_FLAGS
 #undef HAVE_SCTP_PDAPI_EVENT_PDAPI_STREAM
-- 
1.8.3.1

